Docker build doesn't compile static assets

### Describe the bug
v0.7 added a build dependency on go-bindata for including database schema files in the final binary. But when building the Docker image, the static data file isn't generated at the correct time, and the build fails. 

I could use help on this from anyone with more Docker experience, cc @koehn @BenOvermyer @kaiyou @jlelse

### Steps to reproduce (if necessary)
1. Run `make build-docker`
1. Wait a while
1. Get `app.go:276:18: undefined: Asset` _(normally defined in the generated `bindata.go` file)_

### Expected behavior
Image successfully builds

### Application configuration
N/A

**Version or last commit**: v0.7.0

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,25 +1,24 @@
+# Build image
 FROM golang:1.11.2-alpine3.8 as build
 
 RUN apk add --update nodejs nodejs-npm make g++ git sqlite-dev
-RUN npm install -g less
-RUN npm install -g less-plugin-clean-css
+RUN npm install -g less less-plugin-clean-css
 
-WORKDIR /go/src/app
+RUN mkdir -p /go/src/github.com/writeas/writefreely
+WORKDIR /go/src/github.com/writeas/writefreely
 COPY . .
 
-RUN make assets
-RUN make install
-RUN make ui
-RUN make deps
-
+RUN make build \
+ && make ui
 RUN mkdir /stage && \
     cp -R /go/bin \
-       /go/src/app/templates \
-       /go/src/app/static \
-       /go/src/app/pages \
-       /go/src/app/keys \
+      /go/src/github.com/writeas/writefreely/templates \
+      /go/src/github.com/writeas/writefreely/static \
+      /go/src/github.com/writeas/writefreely/pages \
+      /go/src/github.com/writeas/writefreely/keys \
       /stage
 
+# Final image
 FROM alpine:3.8
 
 RUN apk add --no-cache openssl ca-certificates

